src/components/CardSecondary/CardSecondary.tsx

```diff
@@ -63,14 +66,14 @@ const InfoLink = styled.a`
   color: ${({ theme }) => theme.click.card.secondary.color.link.default};
```

Comment:
Add `gap: ${({ theme }) => theme.click.card.secondary.space.link.gap};` here to InfoLink